### ISSUE TITLE
when fine_tune process, which folder is used?

### ISSUE THREAD

**Issue by Johnson-yue** [hologerry/AGIS-Net#9]
Hi, when I fine-tune with skeleton_gray_texture dataset  , it including 3 folder : **style** , **train**, **val** , which meaning of those folder.

**style** folder has: 100 data,
**train** folder has: 639 data,
**val** folder has : 334 data

then fine-tune network ,which data will be used? only **style** or only **train** ?

and the **val** use for test ??

**Comment by hologerry** [comment]
Read the dataset code please...

**Comment by Johnson-yue** [comment]
yes, I try it 
![Screenshot from 2019-10-24 17-54-03](https://user-images.githubusercontent.com/10268274/67474690-76766800-f687-11e9-84c9-ac1f3e6c84bd.png)

I don't understand why you finetune with full **train** dataset 

It means that style data has been trained , so but no new style data genrated!

**Comment by hologerry** [comment]
If you have carefully read the `XXX_dataset.py` and the paper, you will know how we use glyphs with and without ground truth images. We do not use the FULL dataset, instead the content input.
The label handle if these glyphs have ground truth.
https://github.com/hologerry/AGIS-Net/blob/8a744ff6c4f329a95daab0aa4f728d8be3858da4/data/cn_few_fusion_dataset.py#L61
<img width="555" alt="Screen Shot 2019-10-24 at 18 02 30" src="https://user-images.githubusercontent.com/11660168/67475273-788cf680-f688-11e9-8978-797f5b797a56.png">

**Comment by hologerry** [comment]
Why do we even need NEW STYLE data?

**Comment by Johnson-yue** [comment]
oh , you using __label__ to control which image will be used . I know , I will check code again. and 

How I understand 

> Although unseen characters cannot contribute to the L1 loss , they are still useful for the adversarial training, helping us get more satisfactory results

I think **unseen characters** should not be used in training so it not exist! , how it help us get more satisfactory results ?
